include/boost/geometry/strategies/spherical/distance_haversine.hpp

```diff
@@ -124,13 +124,6 @@ on a perfect sphere using haversine
 \tparam CalculationType \tparam_calculation
 \author Adapted from: http://williams.best.vwh.net/avform.htm
 \see http://en.wikipedia.org/wiki/Great-circle_distance
-\note (from Wiki:) The great circle distance d between two
-points with coordinates {lat1,lon1} and {lat2,lon2} is given by:
-    d=acos(sin(lat1)*sin(lat2)+cos(lat1)*cos(lat2)*cos(lon1-lon2))
-A mathematically equivalent formula, which is less subject
-    to rounding error for short distances is:
-    d=2*asin(sqrt((sin((lat1-lat2) / 2))^2
-    + cos(lat1)*cos(lat2)*(sin((lon1-lon2) / 2))^2))
 \qbk{
 [heading See also]
 [link geometry.reference.algorithms.distance.distance_3_with_strategy distance (with strategy)]
```

Comment:
So this was the actual bug, it was missing, but it is back now.

![image](https://github.com/user-attachments/assets/a144a847-c883-4e61-a516-4cfd7543ea90)